src/main/java/frc/robot/commands/BatteryCommand.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.BatterySubsystem;
+
+public class BatteryCommand extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final BatterySubsystem subsystem;
+
+  public BatteryCommand(BatterySubsystem s) {
+    addRequirements(s);
+    subsystem = s;
+  }
+
+  @Override
+  public void initialize() {}
+
+  @Override
+  public void execute() {
+    subsystem.periodic();
```

Comment:
subsystem.periodic s run once per run of the command scheduler already?

https://docs.wpilib.org/en/stable/docs/software/commandbased/subsystems.html

Don't think we need this command.

Reply:
Completed, removed it